docs/mimimal-softlayer.yml

```diff
@@ -1,117 +1,74 @@
-# The following line helps maintain current documentation at http://docs.cloudfoundry.org.
-# code_snippet cf-minimal-softlayer start
 ---
 name: cf
 director_uuid: REPLACE_WITH_DIRECTOR_ID
 
 releases:
 - {name: cf, version: latest}
-- {name: postgres, version: latest}
 - {name: diego, version: latest}
-- {name: etcd, version: latest}
-- {name: garden-linux, version: latest}
+- {name: garden-runc, version: latest}
 - {name: cflinuxfs2-rootfs , version: latest}
 
 networks:
```

Comment:
Interestingly, no, we don't. For some reason, even though cf_private is referenced in 3 other locations, it's not really used. This manifest is tested and works as is. But you're right, it's confusing. I'll try to remove them and test again.